### DISCUSSION TITLE
vue-generate里面将state赋值给新的变量，为什么格式

### DISCUSSION THREAD

**Discussion by youyanping** [opentiny/tiny-engine#383] [Q&A]
![QQ截图20240407174055](https://github.com/opentiny/tiny-engine/assets/16658159/50201c9d-41f9-4501-bc4c-b6abd829eadb)
![QQ截图20240407174106](https://github.com/opentiny/tiny-engine/assets/16658159/76141c83-63e9-4dda-acc8-a494c4ff8bc1)
![QQ截图20240407174115](https://github.com/opentiny/tiny-engine/assets/16658159/cb97f4fd-b9f5-4d98-bfd3-e8187f797da0)

**Comment by youyanping** [comment]
如何变成 data() {
    return {
      dataDisk: [1, 2, 3],
      name: "32121"
    }
  }, 这样

**Reply by chilingling** [reply]
拼接字符串就好了，把 reactiveStatement 对象变成你想要的格式的字符串
@youyanping

**Reply by youyanping** [reply]
![2fb0fc256e7d2d7a4b08f8beea325cb](https://github.com/opentiny/tiny-engine/assets/16658159/57d9e5c6-36de-402c-b124-808b58397467)
发现好像是这边有缓存？这个dist文件怎么生成呢，执行命令没效果

**Reply by chilingling** [reply]
进入到 vue-generator 文件夹，执行 `pnpm build` 命令
